README.md

```diff
@@ -0,0 +1,95 @@
+Uplink PHP client
+=============
+
+Client for decentralized object storage on [Tardigrade.io](https://tardigrade.io/)
+
+[▶ 40 second video](https://www.youtube.com/watch?v=QOjM5ERd8yo&feature=youtu.be)
+
+Requirements
+----------
+
+- Tardigrade API key
+- PHP >= 7.4
+
+Installation
+---------
+
+1. Enable the FFI in php.ini:
+
+```
+extension=ffi
+```
+
+2. (optional) To use it in web request also adjust this setting from "preload" to "true":
+
+```
+ffi.enable=true
+```
+
+3. Install using composer. Copy and run:
+
+```
+composer config repositories.storj/uplink '{
+    "type": "package",
+    "package": {
+        "name": "storj/uplink",
+        "version": "1.0.0",
```

Comment:
What version do we release initially? 1.0.0 or 0.1.0?

Reply:
I would recommend 0.1.0 (or alternatively 1.0.0-rc) for a month... and let people test it out to find things that were missed and that may require breaking api changes.

Reply:
Changed to 0.1.0